p2composite.example.parent/pom.xml

```diff
@@ -21,11 +21,9 @@
   </modules>
 
   <properties>
-    <tycho-version>2.2.0</tycho-version>
-    <tycho-extras-version>${tycho-version}</tycho-extras-version>
-    <maven-antrun-plugin.version>1.7</maven-antrun-plugin.version>
```

Comment:
You forgot to update the antrun plugin. Actually, you completely removed its version here.